plans/placebo/main.go

```diff
@@ -14,8 +16,27 @@ func run(runenv *runtime.RunEnv) error {
 		panic("test case sequence number not set")
 	}
 
-	if runenv.TestCaseSeq != 0 {
+	switch runenv.TestCaseSeq {
+	case 0:
+		return nil
+	case 2:
+		// expose prometheus endpoint
+		listener := runenv.MustExportPrometheus()
+		defer listener.Close()
+
+		// create context for cancelation
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		// snapshot metrics every second and save them into "metrics" directory
+		err := runenv.HTTPPeriodicSnapshots(ctx, listener.Addr().String(), time.Second, "metrics")
+		if err != nil {
+			return err
+		}
+
+		time.Sleep(time.Minute)
```

Comment:
Do we want the placebo test to last so long? Won't it slow down integration tests? Just 10 seconds should be enough to validate it's working?

Reply:
This is a specific placebo test: `placebo/metrics` just to try out the metrics. It's not being run on the integration tests AFAIK.